src/@types/@emotion/hash/index.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '@emotion/hash' {
+  function murmurhash(value: string): string
+  export default murmurhash
```

Comment:
Is this because we used `hash` here: https://github.com/segmentio/ui-box/pull/28/files#diff-1d273bf98dff139d43f337f94d03b0e2R31?

Reply:
It's because:
1. There are no types for `@emotion/hash` and I didn't want to change libs (neither did Roland)
2. I wanted `strict: true` :) which required us to type or ignore that lib. Typing it felt right.